lib/cn/cn_internal.h

```diff
@@ -68,7 +68,7 @@ struct cn {
     u32 cn_cflags;
 
     const char *cn_kvdb_alias;
-    char        cn_kvsname[HSE_KVS_NAME_LEN_MAX];
+    const char *cn_kvs_name;
```

Comment:
We used to support renaming a KVS.  I don't see in `hse.h`.  Did we drop that years ago?

Reply:
What function name are you thinking of?

Reply:
We had a public API or CLI command to rename a KVS.

Reply:
Can't think of anything since I have been around